docs/DeveloperGuide.md

```diff
@@ -1077,33 +1077,408 @@ testers are expected to do more *exploratory* testing.
+
+        - Test case: `ls`<br>
+          Expected: All the groups under root directory will be displayed.
+
+        - Test case: `ls grp-001`<br>
+          Expected: If there is no such a group with GroupId, `grp-001`, then an error message will appear at the output box below the command box. Otherwise, all the students in `grp-001` will be displayed.
+
+        - Test case: `ls x` (where x is an invalid path)<br>
+          Expected: An error message of `Invalid relative path` will be shown.
+
+   1. Condition 2: When the current directory is group directory e.g. `grp-001`
+
+        - Test case: `ls`<br>
+          Expected: All the students under `grp-001` will be displayed.
+
+        - Test case: `ls 0001Y`<br>
+          Expected: An error message of `Cannot show children list for this path` will be shown.
+
+        - Test case: `ls x` (where x is an invalid path)<br>
+          Expected: An error message of `Invalid relative path` will be shown.
+
+### Display all tasks
+
+1. Displays all tasks depending on the directory you are at
+
+    1. Condition 1: When the current directory is root directory,
+
+        - Test case: `cat ~/grp-001`<br>
+          Expected: If there is no such a group with GroupId, `grp-001`, then an error message will appear at the output box below the command box. Otherwise, the task list of `grp-001` will be displayed.
+
+        - Test case: `cat ~/grp-001/0123Y`<br>
+          Expected: If there is no such a student with StudentId, `0123Y`, in `grp-001`, then an error message will appear at the output box below the command box. Otherwise, the task list of `0123Y` will be displayed.
+
+        - Test case: `cat x` (where x is an invalid path)<br>
+          Expected: An error message of `Invalid relative path` will be shown.
+
+        - Other incorrect `cat` commands to try: `cat`<br>
+          Expected: An error message of `Cannot show task list for this path: ~` will be shown.
+
+    1. Condition 2: When the current directory is group directory e.g. `grp-001`,
+
+        - Test case: `cat`<br>
+          Expected: The task list of `grp-001` will be displayed.
+
+        - Test case: `cat 0123Y`<br>
+          Expected: If there is no such a student with StudentId, `0123Y`, in `grp-001`, then an error message will appear at the output box below the command box. Otherwise, the task list of `0123Y` will be displayed.
+
+        - Test case: `cat x` (where x is an invalid path)<br>
+          Expected: An error message of `Invalid relative path` will be shown.
 
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
-    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+    1. Prerequisites: Able to access the `/data` folder (created in the same folder containing the `ProfBook.jar` file).
 
-1. _{ more test cases …​ }_
+    1. Test case: Navigate to the `/data` folder and delete the `profbook.json` file. Then reopen the application.
```

Comment:
I know this is what the other team did but follow the other one, ie the screenshot i send in the telegram group. IE the one where they make some changes using commands and it is reflected in the data file